scss/foundation-styles/type.scss

```diff
@@ -1,5 +1,6 @@
 a {
   color: $dark-blue; //style override
+  text-decoration: none;
```

Comment:
Please separate overrides from `/foundation-styles`. Ideally we want to keep those files untouched so they can represent the copy we "borrowed" from Foundation repo. Our goal is to have few overrides as possible so putting overrides in their own files will make code clean up easier (e.g., we can easily compare the overrides with existing Pulse overrides and see if any clean up work needs to be done instead of introducing more overrides on top). By doing this hopefully we will end up having minimal overrides once the brand update work is done! 🤓

Reply:
Override files are all in #1054, this is now added here https://github.com/mozilla/network-pulse/pull/1054/files#diff-3e5deca6ae7e886bb713417b3db18cd1R1.